### ISSUE TITLE
Broken on Crystal 0.29.0

### ISSUE THREAD

**Issue by vladfaust** [vladfaust/core#106]
```
in spec/bulk_query/delete_spec.cr:8: missing argument: rs
Overloads are:
 - Onyx::SQL::Serializable.new(rs : DB::ResultSet, stop_column : String | ::Nil = nil, column_indexer = ColumnIndexer.new)

    user1 = User.new(uuid: uuid1)
```

Whereas `Onyx::SQL::Serializable.new` is `protected`:

https://github.com/onyxframework/sql/blob/c197a2267328f073bbcaeb9d75b80f45c5dae844/src/onyx-sql/serializable.cr#L190-L194

And `Onyx::SQL::Model.new` is not:

https://github.com/onyxframework/sql/blob/c197a2267328f073bbcaeb9d75b80f45c5dae844/src/onyx-sql/model.cr#L146

**Comment by vladfaust** [comment]
Related: https://github.com/amberframework/granite/issues/337

**Comment by vladfaust** [comment]
Fixed in 5c46c6dacb08d266e2e97793ecaac5b91b1248c9. Kudos to @Blacksmoke16! :tada:
